android/spindance.code-challenge.mobile.android.weather-sensor-service/.gitattributes

```diff
@@ -0,0 +1,6 @@
+#
+# https://help.github.com/articles/dealing-with-line-endings/
+#
+# These are explicitly windows files and should use crlf
+*.bat           text eol=crlf
+
```

Comment:
Shouldn't the `android` directory (where this file is) be pretty much empty, since it's a submodule? This includes the license, .gitignore, readme, gradle files and source code (*.kt) below.

Reply:
OK, maybe I didn't look close enough, sorry. It looks like this si the Kotlin and Swift code you had to write in order to bridge things appropriatey. My bad.

Reply:
... nevertheless, should the readme and license files be present here?

Reply:
When I added the git submodules, it just cloned the entire repo into the android folder. Since these files were included in the android weather sensor repo they are included here.

Reply:
On your local hard drive, the submodule will look like a clone, but in git, it's really just a reference to the submodule. So I wouldn't expect there to be any files from the submodule itself committed in this repo.

Reply:
Oh I see what you are saying, I'll look into this

Reply:
I re-added the modules and now the folders just link to the respective repos which seems must more like the expected behavior